src/main/java/trackitnus/logic/commands/lesson/EditLessonCommand.java

```diff
@@ -29,41 +28,33 @@
     public static final String MESSAGE_USAGE = Lesson.TYPE + " " + COMMAND_WORD
         + ": Edits the details of a lesson."
         + " At least one of the details has to be specified. "
-        + "Parameters: "
-        + PREFIX_CODE + "MODULE_CODE "
-        + PREFIX_TYPE + "TYPE "
-        + "[" + PREFIX_DATE + "DATE] "
-        + "[" + PREFIX_ADDRESS + "ADDRESS] "
-        + "[" + PREFIX_WEIGHTAGE + "WEIGHTAGE]\n"
-        + "Example: " + Lesson.TYPE + " " + COMMAND_WORD + " "
+        + "Parameters: INDEX (must be a positive integer) "
```

Comment:
Maybe remove the `(must be a positive integer)` since you already have the INVALID_INDEX_MESSAGE. To keep it consistent with Task and Module. Same for DeleteLesson Command too